### PR TITLE
md5: add German translation

### PR DESCRIPTION
**Pull request by marciska**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.de/osx/md5.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# md5
+
+> Berechnet kryptografrische MD5-Prüfsummen.
```
Comment (marchersimon) [inline pages.de/osx/md5.md:3]:
```suggestion
> Berechne kryptografrische MD5-Prüfsummen.
```

File: pages.de/osx/md5.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+> Mehr Informationen: <https://ss64.com/osx/md5.html>.
+
+- Berechne die MD5-Prüfsumme einer Datei und gebe sie mit Dateinamen auf der Konsole aus:
```
Comment (marchersimon) [inline pages.de/osx/md5.md:6]:
```suggestion
- Berechne die MD5-Prüfsumme einer Datei und gib sie mit Dateinamen auf der Konsole aus:
```

File: pages.de/osx/md5.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`md5 {{datei_name}}`
+
+- Berechne die MD5-Prüfsumme mehrerer Dateien und gebe sie mit Dateinamen auf der Konsole aus:
```
Comment (marchersimon) [inline pages.de/osx/md5.md:10]:
```suggestion
- Berechne die MD5-Prüfsumme mehrerer Dateien und gib sie mit Dateinamen auf der Konsole aus:
```

File: pages.de/osx/md5.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`md5 {{datei_name_1}} {{datei_name_2}}`
+
+- Gebe NUR die MD5 Prüfsumme auf der Konsole aus:
```
Comment (marchersimon) [inline pages.de/osx/md5.md:14]: I don't think the capitalization is necessary.
```suggestion
- Gib nur die MD5 Prüfsumme auf der Konsole aus:
```

File: pages.de/osx/md5.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`md5 -q {{datei_name}}`
+
+- Berechne die MD5-Prüfsumme für den gegebenen Text (keine Datei!):
```
Comment (marchersimon) [inline pages.de/osx/md5.md:18]: I also don't think this is necessary to add. It's not even in the english page.
```suggestion
- Berechne die MD5-Prüfsumme für den gegebenen Text:
```

**Comment by marciska** [comment]
I agree with the changes. Thanks!

File: pages.de/osx/md5.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# md5
+
+> Berechne kryptografrische MD5-Prüfsummen.
```
Comment (pixelcmtd) [inline pages.de/osx/md5.md:3]:
```suggestion
> Berechne kryptografische MD5-Prüfsummen.
```

File: pages.de/osx/md5.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+- Berechne die MD5-Prüfsumme einer Datei und gib sie mit Dateinamen auf der Konsole aus:
+
+`md5 {{datei_name}}`
+
+- Berechne die MD5-Prüfsumme mehrerer Dateien und gib sie mit Dateinamen auf der Konsole aus:
+
+`md5 {{datei_name_1}} {{datei_name_2}}`
+
+- Gib nur die MD5 Prüfsumme auf der Konsole aus:
```
Comment (pixelcmtd) [inline pages.de/osx/md5.md:14]:
```suggestion
- Gib nur die MD5-Prüfsumme auf der Konsole aus:
```

**Comment by marciska** [comment]
Fixed smaller mistakes pointed out.

**Review by pixelcmtd — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**